fe1-web/src/features/evoting/screens/CreateElection.tsx

```diff
@@ -302,7 +304,7 @@ const CreateElection = () => {
           {STRINGS.event_creation_must_be_connected}
         </Text>
       )}
-      {electionName === '' && (
+      {electionName.trim() === '' && (
```

Comment:
Yes, this is great! If something is off usually it is best to tell the user and let them fix it. Trimming whitespace is fine without prompting the user is fine as it does not change the meaning but having an empty election name is something they have to fix.

Reply:
Okay you have to do trimming here again, I think having a `trimmedElectionName` would definitely be a good idea 😅